.eslintrc.js

```diff
@@ -1,30 +1,39 @@
 module.exports = {
-  extends: 'erb',
+  extends: [
+    "erb",
+    "airbnb/hooks",
+    "airbnb-typescript",
+    "eslint:recommended",
+    "plugin:jest/recommended",
+    "plugin:prettier/recommended",
```

Comment:
Hi Tom! 
Several q's for ya. 
1. is this the ideal order of 'extends'?
usually i see 'prettier' last
2. is plugin: prettier/recommended interchangeable with 'prettier' ?

Reply:
1. I didn't think the order mattered, but when I checked the docs, it says to load it last. I pushed an update
https://github.com/prettier/eslint-plugin-prettier

2. According to the doc (step 2) in the link above, this is how to load it. It shows what eslint expands it to
https://github.com/prettier/eslint-plugin-prettier

I googled eslint-plugin-prettier vs prettier-eslint. It looks like there are multiple ways to do it, but plugin:prettier/recommended seems to be recommended

Reply:
Take at look at my new pushes, I found some other things that needed updating too

Reply:
@jbeanz can you mark this as resolved?